### PR TITLE
Add UDP broadcasting

### PR DESCRIPTION
**Pull request by tlunter**
Based off of #15

Adds a simple resque job that will run when the /broadcast/ endpoint is hit

### CODE REVIEW

**Comment by tlunter** [comment]
@nahiluhmot @djosborne

File: Rakefile [hunk 1]
```diff
@@ -22,11 +23,20 @@ namespace :db do
...
 end
 
+desc 'Swaps redis with the container name'
+task swap_redis_if_needed: :app_environment do
+  if ENV['DOCKER']
+    Resque.redis = Redis.new(host: 'redis')
+  end
+end
+
+task environment: ['db:load_config', :app_environment, :swap_redis_if_needed]
```
Comment (nahiluhmot) [inline Rakefile]: I like this.

File: app/cappy/services/broadcast.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    module Broadcast
+      @queue = :cappy
+      BROADCAST_ADDRESS = '255.255.255.255'
+      BROADCAST_PORT = 10_000
+
+      module_function
+
+      def perform
+        puts 'Starting CCS broadcast'
+        5.times do
```
Comment (nahiluhmot) [inline app/cappy/services/broadcast.rb]: Why five times?

Comment (Daniel0524) [inline app/cappy/services/broadcast.rb]: Because UDP is a pretty devil may care protocol?

Comment (tlunter) [inline app/cappy/services/broadcast.rb]: That and to allow for timing between hitting the button and getting the device on or whatever.

Comment (nahiluhmot) [inline app/cappy/services/broadcast.rb]: Word, just wondering how ya arrived on five. Makes sense though.

**Pull request merged**